app/models/response.rb

```diff
@@ -1,6 +1,27 @@
 class Response < ApplicationRecord
+  include AASM
+  
+  default_scope { order(created_at: :desc) }
```

Comment:
Just noticed this. Instead of using `reverse_order` above I would just suggest changing the order to ascending instead.

```suggestion
  default_scope { order(:created_at) }
```
(default is ascending on order)